tests/integration/operatorhub/cmd_link_test.go

```diff
@@ -0,0 +1,93 @@
+package integration
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo link command tests for OperatorHub", func() {
+
+	var commonVar helper.CommonVar
+	var oc helper.OcRunner
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+		oc = helper.NewOcRunner("oc")
+	})
+
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+
+	Context("Operators are installed in the cluster", func() {
+
+		var etcdOperator string
+		var etcdCluster string
+
+		BeforeEach(func() {
+			// wait till odo can see that all operators installed by setup script in the namespace
+			odoArgs := []string{"catalog", "list", "services"}
+			operators := []string{"etcdoperator", "service-binding-operator"}
+			for _, operator := range operators {
+				helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
+					return strings.Contains(output, operator)
+				})
+			}
+
+			list := helper.CmdShouldPass("odo", "catalog", "list", "services")
```

Comment:
Can we use the new runner functions here.
Functions mentioned in this [PR](https://github.com/openshift/odo/pull/4835) will remove be removed.
I would request to use those instead of the old ones.

Reply:
@feloy can you also make changes mentioned in this comment.  This [PR](https://github.com/openshift/odo/pull/4835) will remove the older cmd helper functions.
Thanks